_toc.yml

```diff
@@ -1,5 +1,59 @@
 - file: index
 - chapters:
+  - file: tutorials/start_index
+    sections:
+    - file: tutorials/fundamentals/installation
+    # - file: tutorials/absolute_beginners
+    - file: tutorials/fundamentals/getting_started
+    - file: plugins/stable/find-and-install-plugin
+    - file: tutorials/fundamentals/viewer
+    # The following will be replaced by
+    # - file: tutorials/fundamentals/layers/index
+    - file: tutorials/fundamentals/image
+    - file: tutorials/fundamentals/labels
+    - file: tutorials/fundamentals/points
+    - file: tutorials/fundamentals/shapes
+    - file: tutorials/fundamentals/surface
+    - file: tutorials/fundamentals/tracks
+    - file: tutorials/fundamentals/vectors
+    title: Getting started
```

Comment:
This title belongs with the `tutorials/start_index` above, not the line immediately below it, right?

If this is also valid yaml, for ease of reading can we put the section titles up the top for each bit like this?
```yaml
- chapters:
  - file: tutorials/start_index
    title: Getting started
    sections:
    - file: tutorials/fundamentals/installation
    - ... long list
  - file: tutorials/index
    title: Tutorials
    sections:
    - file: tutorials/segmentation/index
    - ... long list
  - file: howtos_index
    title: How-to guides
    sections:
    # - file: tutorials/fundamentals/layers/index
    - file: tutorials/applications/napari_imageJ
    - ... long list
```